connector/protobuf/src/main/scala/org/apache/spark/sql/protobuf/utils/SchemaConverters.scala

```diff
@@ -67,9 +68,23 @@ object SchemaConverters extends Logging {
       existingRecordNames: Map[String, Int],
       protobufOptions: ProtobufOptions): Option[StructField] = {
     import com.google.protobuf.Descriptors.FieldDescriptor.JavaType._
+
+    val upcastUnsignedInts = protobufOptions.upcastUnsignedInts
```

Comment:
can use `protobufOptions.upcastUnsignedInts` directly below. 
Or add 'option' to this variable's name. It is more readable.

Reply:
sure, i've inlined it 👍